src/openthaigpt_pretraining/utils/data_cleaning.py

```diff
@@ -0,0 +1,10 @@
+
+
+def clean_text(line: str) -> str:
+    # () is remainder after link in it filtered out
+    return line.strip().replace("\n", " ").replace("()", "")
+
+def filter_out_line(line: str) -> str:
+    if len(line) < 80:
```

Comment:
I think you should change 80 to function argument. We can adjust this number later.